src/components/PhoneNumberInput/styles.scss

```diff
@@ -36,6 +36,19 @@
         &:hover, &.selected {
             background-color: var(--background-highlight-color);
         }
+
+        .sg-country-list-item-label {
+            display: flex;
+            align-items: center;
+            justify-content: space-between;
+            flex: 1;
```

Comment:
is this required?

Reply:
Design team wanted to update the way the component displays country name and its code. For ref, PFB the existing design and requirements.

<img width="1247" alt="Screenshot 2023-09-13 at 12 53 04 PM" src="https://github.com/contacto-io/style-guide/assets/136322627/59584570-7fe8-4be4-8a37-5134cd22a2dc">